google/cloud/aiplatform/training_jobs.py

```diff
@@ -3720,7 +3748,11 @@ def run(
         forecast_horizon: int,
         data_granularity_unit: str,
         data_granularity_count: int,
+        training_fraction_split: Optional[float] = None,
+        validation_fraction_split: Optional[float] = None,
+        test_fraction_split: Optional[float] = None,
         predefined_split_column_name: Optional[str] = None,
+        timestamp_split_column_name: Optional[str] = None,
```

Comment:
What was the previous behavior before adding these split columns? Did it split on the time_series_identifier_column?

Reply:
@thehardikv do you know?

Reply:
It defaults to fraction split.